elide-core/src/main/java/com/yahoo/elide/core/datastore/inmemory/InMemoryStoreTransaction.java

```diff
@@ -39,6 +42,8 @@
  */
 public class InMemoryStoreTransaction implements DataStoreTransaction {
```

Comment:
Can we make `InMemoryStoreTransaction` extends `TransactionWrapper`?

Reply:
I tried, but I'm not sure it can be done.

Reply:
I originally tried this, but had trouble delegating a method call to DataStoreTransaction.  It has to call the default loadObject method in DataStoreTransaction.  Maybe there is a way to do that - but I couldn't get it to work.